gcp_variant_transforms/pipeline_common.py

```diff
@@ -199,6 +200,45 @@ def read_headers(pipeline, pipeline_mode, all_patterns):
   return headers
 
 
+def read_variants(pipeline,  # type: beam.Pipeline
+                  all_patterns,  # type: List[str]
+                  pipeline_mode,  # type: int
+                  allow_malformed_records,  # type: bool
+                  representative_header_lines=None,  # type: List[str]
+                  vcf_parser=vcfio.VcfParserType.PYVCF  # type: int
```

Comment:
Umm, I know the underlying value of the enum is int, but shouldn't the type of this argument still be VcfParserType? not sure about convention here.

Reply:
Done. Replaced with PipelineModes.

Reply:
Also changed the vcf_parser type.